src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -194,12 +231,12 @@ object HeaderPlugin extends AutoPlugin {
             groupedFiles.flatMap(checkHeader(fileType, commentStyle))
         }
     if (filesWithoutHeader.nonEmpty)
-      log.error(
+      throw new MessageOnlyException(
         s"""|There are files without headers!
-            |  ${filesWithoutHeader.mkString(s"$newLine  ")}
-            |""".stripMargin
+              |  ${filesWithoutHeader.mkString(s"$newLine  ")}
```

Comment:
Was that additional indentation intended?

Reply:
Yuck! I blame IntelliJ or ScalaFMT, or both.